firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -66,10 +66,12 @@ static boolean isValidApiKeyFormat(@Nullable String apiKey) {
     return API_KEY_FORMAT.matcher(apiKey).matches();
   }
 
-  /* Returns a random number of milliseconds less than or equal to 1000. This helps to avoid cases
-   where many clients get synchronized by some situation and all retry at once, sending requests
-   in synchronized waves. The value of random_number_milliseconds is recalculated after each retry
-  request.*/
+  /**
+   * Returns a random number of milliseconds less than or equal to 1000. This helps to avoid cases
+   * where many clients get synchronized by some situation and all retry at once, sending requests
+   * in synchronized waves. The value of random_number_milliseconds is recalculated after each retry
+   * request.
+   */
   public long getRandomMillis() {
     // Mockito doesn't allow to mock static methods. As a result this util method is not static.
     return (long) (Math.random() * 1000);
```

Comment:
Just for completeness:
A random number generator should, similar to a timesource, be injected into code to allow deterministic testing of code.

No action item! I just wanted to shared this with you.

Reply:
I like it. Thanks for the suggestion. I added a TODO and will follow up the change in a separate PR.